src/mockServer/routes/diseases.js

```diff
@@ -1,22 +1,38 @@
-import { diseasesDTO } from "../fixtures/diseasesDTO";
-
-export const diseasesRoutes = (server) => {
-  server.namespace("/diseases", () => {
-    server.get("/all").intercept((_req, res) => {
-      res.status(200).json(diseasesDTO);
-    });
-    server.get("/opd").intercept((_req, res) => {
-      res.status(200).json(diseasesDTO.filter(({ opdInclude }) => opdInclude));
-    });
-    server.get("/ipd/in").intercept((_req, res) => {
-      res
-        .status(200)
-        .json(diseasesDTO.filter(({ ipdInInclude }) => ipdInInclude));
-    });
-    server.get("/ipd/out").intercept((_req, res) => {
-      res
-        .status(200)
-        .json(diseasesDTO.filter(({ ipdOutInclude }) => ipdOutInclude));
-    });
-  });
-};
+import { diseasesDTO } from "../fixtures/diseasesDTO";
+
+export const diseasesRoutes = (server) => {
+  server.namespace("/diseases", () => {
+    server.get("/all").intercept((req, res) => {
+      res.status(200).json(diseasesDTO);
+    });
+    server.get("/opd").intercept((req, res) => {
+      res.status(200).json(diseasesDTO);
+    });
+    server.get("/ipd/in").intercept((req, res) => {
+      res.status(200).json(diseasesDTO);
+    });
+    server.get("/ipd/out").intercept((req, res) => {
+      res.status(200).json(diseasesDTO);
+    });

```

Comment:
Nitpick: What about filtering here to have more realistic results (and easier to test and debug)
```suggestion
    server.get("/opd").intercept((req, res) => {
      res.status(200).json(diseasesDTO.filter(({opdInclude}) => opdInclude);
    });
    server.get("/ipd/in").intercept((req, res) => {
      res.status(200).json(diseasesDTO.filter(({ipdInInclude}) => ipdInInclude));
    });
    server.get("/ipd/out").intercept((req, res) => {
      res.status(200).json(diseasesDTO.filter(({ipdOutInclude}) => ipdOutInclude));
    });
```

Reply:
Looks  better.